pkg/txn/storage/txn/bvt/bvt.go

```diff
@@ -0,0 +1,236 @@
+// copyright 2022 matrix origin
```

Comment:
Wait -- how do you use this?   How do test team use this?

Reply:
I just run `go run bvt.go` to start a server. This program does not use the distributed transaction framework to bypass some bugs. After the mem engine passes the bvt test, this program will not be needed anymore. It's temporary. The test team will use the mo-service to run the test.